seed/static/seed/js/controllers/inventory_detail_meters_controller.js

```diff
@@ -162,18 +162,9 @@ angular.module('BE.seed.controller.inventory_detail_meters', [])
       var filterByMeterLabels = function filterByMeterLabels (readings, columnDefs, meterLabels) {
         var timeColumns = ['start_time', 'end_time', 'month', 'year'];
         var selectedColumns = meterLabels.concat(timeColumns);
-        var filteredReadings = readings.map(function (reading) {
-          return Object.entries(reading).reduce(function (newReading, _ref) {
-            var key = _ref[0],
-              value = _ref[1];
-
-            if (selectedColumns.includes(key)) {
-              newReading[key] = value;
-            }
```

Comment:
Because the `timeColumns` are included in `selectedColumns`, this function will return every row, while removing unselected columns.